Cargo.toml

```diff
@@ -19,6 +19,7 @@ ansi_term = "0.11"
 console = "0.6"
 directories = "1.0"
 lazy_static = "1.0"
+libz-sys = "1.0.20"
```

Comment:
Why is this necessary?

Reply:
When i was testing the build for the first time it was failing with `error: failed to run custom build command for libz-sys v1.0.20`. But right now i checked again and it takes care of it implicitly.